net-misc/oidentd/oidentd-3.0.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+DESCRIPTION="Another (RFC1413 compliant) ident daemon"
+HOMEPAGE="https://oidentd.janikrabe.com/"
+SRC_URI="https://files.janikrabe.com/pub/${PN}/releases/${PV}/${P}.tar.xz"
+
+LICENSE="BSD-2 GPL-2 LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="debug ipv6 masquerade selinux"
+
+DEPEND="masquerade? ( net-libs/libnetfilter_conntrack )"
+
+RDEPEND="
+	acct-user/oidentd
+	acct-group/oidentd
+	selinux? ( sec-policy/selinux-oident )
+	${DEPEND}"
+
+src_prepare() {
+	sed -i '/ExecStart/ s|$| -u oidentd -g oidentd|' contrib/systemd/*.service || die
```

Comment:
```
 * QA: following sed did not cause any changes
 *     sed -i '/ExecStart/ s|$| -u oidentd -g oidentd|' contrib/systemd/*.service || die;
```

Reply:
He, why I didn't get that QA warning during compilation?

Reply:
Custom tool: https://github.com/ionenwks/iwdevtools

Reply:
It's from iwdevtools - https://packages.gentoo.org/packages/app-portage/iwdevtools still betatesting this though, but if it gives such an error, it's nice to ask for a confirmation whether it does anything or not. 

Now I can see the asterisk in here either not matching anything, or it getting missed by ionen's qa-sed.

Reply:
Good to know, thanks!

Reply:
I think, this warning is a false positive? After installing `iwdevtools` I am getting this warning, but if I do run manally the command, only without `|| die `, all *.service files have been changed correctly?

Reply:
All files in /var/tmp/portage are also correctly.